aea/helpers/transaction/base.py

```diff
@@ -33,6 +33,68 @@
 Address = str
 
 
+class Kwargs:
```

Comment:
Why does this need to be added? this should already exist somewhere...

Reply:
This is exist on the packages, we cannot import from packages to aea